core/manager/supportingservice_manager.go

```diff
@@ -50,7 +50,7 @@ func NewKogitoSupportingServiceManager(context operator.Context, supportingServi
 }
 
 func (k kogitoSupportingServiceManager) EnsureSingletonService(namespace string, resourceType api.ServiceType) error {
-	k.Log.Info("Ensuring only single instance of supporting service exists")
+	k.Log.Info("Ensuring only single instance of supporting service exists", "resourceType", resourceType)
```

Comment:
We should start investigating [admission controllers](https://book.kubebuilder.io/cronjob-tutorial/webhook-implementation.html) instead of these checks.

Reply:
sure Zanini